README.md

```diff
@@ -4,52 +4,50 @@
 
-## Usage
+## Installation
 
-Start by creating a new client:
+Get the code with:
 
-```go
-client, err := statsd.New("127.0.0.1:8125",
-    statsd.WithNamespace("flubber."),               // prefix every metric with the app name
-    statsd.WithTags([]string{"region:us-east-1a"}), // send the EC2 availability zone as a tag with every metric
-    // add more options here...
-)
-if err != nil {
-    log.Fatal(err)
-}
+```shell
+$ go get github.com/DataDog/datadog-go/statsd
 ```
 
-You can find a list of all the available options [here](https://godoc.org/github.com/DataDog/datadog-go/statsd#Option).
-
-After the client is created, you can start sending metrics:
+Then create a new DogStatsD client:
 
 ```go
-client.Gauge("kafka.health", 1, []string{"env:production", "partition:1", "partition:2"}, 1)
-```
+package main
 
-Each metric call requires the same parameters:
+import (
+    "log"
+    "github.com/DataDog/datadog-go/statsd"
+)
 
-- `name (string)`: The metric name that will show up in Datadog
-- `value`: The value of the metric. Type depends on the metric type
-- `tags ([]string)`: The list of tags to apply to the metric. Multiple tags can have the same key
-- `rate (float)`: The sampling rate in `[0,1]`. For example `0.5` means that half the calls will result in a metric being sent to Datadog. Set to `1` to disable sampling
+func main() {
+    statsd, err: = statsd.New("127.0.0.1:8125")
+    if err != nil {
+        log.Fatal(err)
+    }
+}
+```
 
-You can find all the available functions to report metrics [here](https://godoc.org/github.com/DataDog/datadog-go/statsd#Client).
+Find a list of all the available options for your DogStatsD Client in the [Datadog-go godoc documentation](https://godoc.org/github.com/DataDog/datadog-go/statsd#Option) or in [Datadog public DogStatsD documentation](https://docs.datadoghq.com/developers/dogstatsd/?tab=go#client-instantiation-parameters).
 
-## Supported environment variables
+### Supported environment variables
 
-- The client can use the `DD_AGENT_HOST` and (optionally) the `DD_DOGSTATSD_PORT` environment variables to build the target address if the `addr` parameter is empty.
-- If the `DD_ENTITY_ID` environment variable is found, its value will be injected as a global `dd.internal.entity_id` tag. This tag will be used by the Datadog Agent to insert container tags to the metrics. You should only `append` to the `c.Tags` slice to avoid overwriting this global tag.
+* The client can use the `DD_AGENT_HOST` and (optionally) the `DD_DOGSTATSD_PORT` environment variables to build the target address if the `addr` parameter is empty.
+* If the `DD_ENTITY_ID` environment variable is found, its value will be injected as a global `dd.internal.entity_id` tag. This tag will be used by the Datadog Agent to insert container tags to the metrics. You should only `append` to the `c.Tags` slice to avoid overwriting this global tag.
```

Comment:
```suggestion
* If the `DD_ENTITY_ID` environment variable is found, its value is injected as a global `dd.internal.entity_id` tag. The Datadog Agent uses this tag to insert container tags into the metrics. To avoid overwriting this global tag, only `append` to the `c.Tags` slice.
```